examples/idempotency/src/app_disabling_idempotency_utility.py

```diff
@@ -0,0 +1,17 @@
+from aws_lambda_powertools.utilities.idempotency import (
```

Comment:
is that for the testing section? we should have the name closely aligned with the section so it's easier to find.

Reply:
@heitorlessa Make sense moving this to specific folder and make changes in the name. Please, check it now.